snuba/datasets/readiness_state.py

```diff
@@ -1,8 +1,21 @@
+from __future__ import annotations
+
 from enum import Enum
+from typing import Any
 
 
 class ReadinessState(Enum):
-    LIMITED = "limited"
-    DEPRECATE = "deprecate"
-    PARTIAL = "partial"
-    COMPLETE = "complete"
+    LIMITED = "limited", 1
```

Comment:
Can we get away with not having level here? Sometimes we want to do enum comparison with strings e.g `x.state == `limited` . This will make it harder since you have to remember the level. If level is already bound to the enum string I see no point in keeping the enum as a tuple.

Reply:
Makes sense. Addressed in comment above.